categories.js

```diff
@@ -113,18 +110,8 @@ exports.normalizeCategory = (normalizedGame, category) => {
 // this function is so dumb
 exports.normalizeLevel = (normalizedGame, level) => {
     level = level.toLowerCase()
-            .replace(new RegExp("'", 'g'), "")
-            .replace(new RegExp("-", 'g'), "")
-            .replace(new RegExp(":", 'g'), "")
-            .replace(new RegExp("\\?", 'g'), "")
-            .replace(new RegExp("&", 'g'), "and")
-            .replace(new RegExp("!", 'g'), "")
-            .replace(new RegExp(",", 'g'), "")
-            .replace(new RegExp("\\.", 'g'), "")
-            .replace(new RegExp("\\(", 'g'), "")
-            .replace(new RegExp("\\)", 'g'), "")
-            .replace(new RegExp(" ", 'g'), "")
-            .replace(new RegExp("the", 'g'), "");
+            .replace(/\W|the/g, "")
```

Comment:
This removes `%`, so I think you can remove the `die%` and `styrofoam%` conditions and just stick with matching `die` and `styrofoam` below.